src/delimiter.js

```diff
@@ -326,10 +326,10 @@ const sqrtSvg = function(sqrtName, height, viewBoxHeight, options) {
         // to a point near (14 units) the bottom of the surd. The viniculum
         // is 40 units thick. So the length of the line in question is:
         const vertSegment = viewBoxHeight - 54;
-        alternate = `M702 0H400000v40H742v${vertSegment}l-4 4-4 4c-.667.667
+        alternate = `M702 80H400000v40H742v${vertSegment}l-4 4-4 4c-.667.667
 -2 1.5-4 2.5s-4.167 1.833-6.5 2.5-5.5 1-9.5 1h-12l-28-84c-16.667-52-96.667
 -294.333-240-727l-212 -643 -85 170c-4-3.333-8.333-7.667-13 -13l-13-13l77-155
- 77-156c66 199.333 139 419.667 219 661 l218 661zM702 0H400000v40H742z`;
+ 77-156c66 199.333 139 419.667 219 661 l218 661zM702 80H400000v40H742z`;
```

Comment:
If we have `const delimiterPadding = 80;` replace the `80` in the this string with `${delimiterPadding}`... or at the very least add a component stating what the 80 is so that if people need to change the padding in the future it's easy to find all the places that need updating.